### ISSUE TITLE
Experiment with constant learning rate with transformers

### ISSUE THREAD

**Issue by cwmeijer** [spokenlanguage/platalea#24]
Do a couple of runs with constant learning, varied systematically between runs, and find out if we can find some learning rate that consistently reduces loss. After that we can experiment with more complicated schemes.

**Comment by egpbos** [comment]
I started a run, [effortless-cosmos-19](https://wandb.ai/spokenlanguage/platalea_transformer/runs/h5wu07p5), with the cyclic scheduler with maximum and minimum set to values that may be more sane for this model (based on [this article](https://towardsdatascience.com/adaptive-and-cyclical-learning-rates-using-pytorch-2bf904d18dee)). So far, it is at least outperforming all previous runs. Will let it run until Monday 8 a.m. (according to my estimation) and then we'll see how it went.

Already, now, one interesting feature is the correlation of the periodic pattern in the loss with the learning rate. This might suggest that, as also mentioned in [the article](https://towardsdatascience.com/adaptive-and-cyclical-learning-rates-using-pytorch-2bf904d18dee), some kind of decreasing maximum might be helpful to press down the big (seemingly useless) peaks in the loss and make it converge even faster.

**Comment by egpbos** [comment]
New run [dainty-dawn-20](https://wandb.ai/spokenlanguage/platalea_transformer/runs/31ivn5al) also logs validation loss on wandb.

**Comment by egpbos** [comment]
I think we can close this, since the training loss is very effectively reduced now. Next problem is validation loss, issue #30.
